reactive_graph/src/computed/async_derived/mod.rs

```diff
@@ -25,6 +25,7 @@ pin_project! {
     pub struct ScopedFuture<Fut> {
         pub owner: Owner,
         pub observer: Option<AnySubscriber>,
+        diagnostics: bool,
```

Comment:
Should these be marked `#[cfg(any(debug_assertions, leptos_debuginfo))]`? I think otherwise diagnostics are turned off anyway.